src/js/utils/ui.js

```diff
@@ -348,6 +348,10 @@ const eventRegisters = {
 };
 
 export function getElementWindow(element) {
+    if (!element) {
+        return;
```

Comment:
If this does not return anything then there will be another exception with the code that uses the `undefined` result. Please see what where we are in the debug code after this point, and don't use the harness - use the events page (no jQuery).

Once you've finished investigating, `return window` here.

Reply:
@robwalch Thanks for the suggestion on using the event page. I was able to pinpoint the issue to being when IE11 and Edge fire pointer events. In those browsers, the `destroy()` method in `dismissButton.js` (in commercial) is setting the element to `null` before the `interactEndHandler` can run. So by the time it does run, the element is `null`.

I updated to make sure `ui.js` returns `window`.

Reply:
This sounds like a bug in `ui.off`. If we call `removeHandlers(this, WINDOW_GROUP)` in `destroy() => this.off()` then `interactEndHandler` should not get triggered (or is it already running?)

```js
    off(name, callback, context) {
        if (!name) {
            removeHandlers(this, WINDOW_GROUP);
            const { handlers } = this;
            Object.keys(handlers).forEach(triggerName => {
                removeHandlers(this, triggerName);
            });
        } else if (eventsApi(name)) {
            removeHandlers(this, name);
        }
        return super.off(name, callback, context);
    }
```